sdks/python/apache_beam/utils/windowed_value.py

```diff
@@ -40,9 +141,12 @@ class WindowedValue(object):
     timestamp: Timestamp associated with the value as seconds since Unix epoch.
     windows: A set (iterable) of window objects for the value. The window
       object are descendants of the BoundedWindow class.
+    pane_info: A PaneInfo descriptor describing the triggering information for
+      the pane that contained this value.  If None, will be set to
+      PANE_INFO_UNKNOWN.
   """
 
-  def __init__(self, value, timestamp, windows):
+  def __init__(self, value, timestamp, windows, pane_info=None):
```

Comment:
Robert, can you comment on whether this extra kwarg is suitable, performance-wise?

Reply:
It should be fine, as long as it's not usually passed as a kwarg. (Also, most of the time we create these with `.with_value` which is more optimized.)